weather/www/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    <title></title>
+
+    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+
+    <!-- IF using Sass (run gulp sass first), then uncomment below and remove the CSS includes above
+    <link href="css/ionic.app.css" rel="stylesheet">
+    -->
+
+    <!-- ionic/angularjs js -->
+    <script src="lib/ionic/js/ionic.bundle.js"></script>
+	<script src="lib/ionic/js/angular/angular-resource.js"></script>
+
+    <!-- cordova script (this will be a 404 during development) -->
+    <script src="cordova.js"></script>
+
+    <!-- your app's js -->
+    <script src="js/app.js"></script>
+	<script src="js/weatherController.js"></script>
+	<script src="js/services.js"></script>
+	<script src="js/constants.js"></script>
+  </head>
+  
+  <!--Header stuff -->
+  <body ng-app="weather">
+	<ion-nav-view class="background">
+	<ion-header-bar class="bar-calm">
+		<h1 class="title">Wenham, MA</h1>
+	</ion-header-bar>
+	
+	<!--Get information for the current day -->
+    <ion-pane ng-controller="weatherCtrl">
+        <ion-content scroll="false" class="has-header">
+          <h2 class="currDay">{{findDays[0]}}</h2>
+		  <div class="row row-67">
+		  <script>console.log("today")</script>
+            <div class="col">
+              Currently<br>
+              {{tempCurrent}} °F </div>
+			  
+			<div class="col">Tonight<br>
+			  {{tempTonightLow}} °F</div>
+			
+			<div class="col">Precipitation<br>
+			  {{todayPrecip}}</div>
+		  
+		<div class="col">Wind<br>
+				  {{todayWind}}</div>
+          </div>
+			
+		<div class="weatherPic">
+			<a class="picItem item-thumbnail-left" href="#">
+				<img src={{getPicture}}>
+			</a>
+		</div>
+		  
+		  <hr>
+		 
+		 <!--Dropdown menu for weather for rest of week -->
```

Comment:
I really liked the drop down menu - It was a nice UI feature. Not sure if it is possible, but maybe it could sort of drop up so that the screen does not get any longer.